extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -183,15 +181,14 @@ class AmpYoutube extends AMP.BaseElement {
     this.element.appendChild(iframe);
 
     this.iframe_ = iframe;
-
     this.playerReadyPromise_ = new Promise(resolve => {
       this.playerReadyResolver_ = resolve;
     });
 
-    this.win.addEventListener(
+    const ampdoc = this.getAmpDoc();
+    ampdoc.win.addEventListener(
```

Comment:
cc/ @aghassemi as he has another PR that touches this path - maybe can update the listener to use the helpers